java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1774,26 +1773,25 @@ public T bzmpop(double timeout, @NonNull String[] keys, @NonNull ScoreModifier m
 
     // TODO add @link to ZMPOP when implemented
     /**
-     * Blocks the connection until one or more elements, that are member-score pair, from the first
-     * non-empty sorted set in the provided list of key names.<br>
-     * <code>BZMPOP</code> is the blocking variant of <code>zmpop</code>.<br>
-     * The members with the highest/lowest score are popped from first sorted set that is non-empty,
-     * with the given <code>keys</code> being checked in the order that they are given.
+     * Blocks the connection unless it pops multiple elements, that is a member-score pair, from the
+     * first non-empty sorted set in the provided list of key names.<br>
```

Comment:
The intention of this is a bit clearer, but it still doesn't make sense. Is this what you meant?
```suggestion
     * Blocks the connection until it pops one or more member-score pairs from the 
     * first non-empty sorted set in the provided list of key names.<br>
```